src/lumigo_tracer/spans_container.py

```diff
@@ -226,6 +226,12 @@ def end(self, ret_val=None) -> Optional[int]:
                     "The lambda will probably fail due to bad return value. " + suffix,
                 )
         self.function_span.update({"return_value": parsed_ret_val})
+        if _is_span_has_error(self.function_span):
+            self.function_span["envs"] = _get_envs_for_span(has_error=True)
+            if event:
+                self.function_span["event"] = EventDumper.dump_event(
+                    copy.deepcopy(event), has_error=True
+                )
         spans_contain_errors: bool = any(
             _is_span_has_error(s) for s in self.spans + [self.function_span]
         )
```

Comment:
I think that `lumigo_dumps` will trim your string again.
please check it E2E. I don't think that it works...

Reply:
It works :)
![image](https://user-images.githubusercontent.com/49228804/96361812-b0ab0600-1131-11eb-9391-60132b0ee77f.png)
![image](https://user-images.githubusercontent.com/49228804/96361820-bc96c800-1131-11eb-8c2a-325ef61668f3.png)

Where is the `lumigo_dumps` that might trim my string?